src/components/button/button_empty/_button_empty.scss

```diff
@@ -96,6 +96,7 @@ $ouiButtonEmptyTypes: (
       // Ghost is unique and ALWAYS sits against a dark background.
       color: $color;
     } @else if ($name == 'text') {
+      line-height: inherit;
```

Comment:
Is the goal for empty buttons to be aligned with other text? How does this impact alignment with other buttons?

Reply:
Makes sense, updated to parent class `ouiButtonEmpty`. Also, this only impacts ouiButtonEmpty which brings this component inline with other button icons: 
<img width="854" alt="Screenshot 2024-08-14 at 11 21 46 AM" src="https://github.com/user-attachments/assets/a27275a9-7614-420d-acec-84559c291360">
<img width="294" alt="Screenshot 2024-08-14 at 11 21 53 AM" src="https://github.com/user-attachments/assets/9bbff132-1ce4-4460-8e8e-2ed9d95ac07f">
<img width="869" alt="Screenshot 2024-08-14 at 11 22 05 AM" src="https://github.com/user-attachments/assets/7ecdf21e-24db-469e-b457-5e6be38a62a9">